test/helpers/data/app-court-reports-helper.js

```diff
@@ -0,0 +1,38 @@
+const knex = require('../../../knex').appSchema
+var Promise = require('bluebird').Promise
+const workloadOwnerHelper = require('./app-workload-owner-helper')
+
+module.exports.insertDependencies = function (inserts) {
+  return knex('court_reports_calculations').first('id')
+  .then(function (crcCalculationId) {
+    return knex('workload_report').whereNull('effective_to').first('id')
+    .then(function (workloadReportId) {
+      return knex('court_reports').max('staging_id AS maxId')
+      .then(function (maxStagingId) {
+        return workloadOwnerHelper.insertDependencies(inserts)
+        .then(function (inserts) {
+          var newCourtReport = {
+            workload_report_id: workloadReportId.id,
+            staging_id: maxStagingId[0].maxId + 1,
```

Comment:
Is this `maxStagingId` necessary? Since we are always querying on a combination of `staging_id` and `workload_report_id` and we are inserting a `workload_report_id` as part of this

Reply:
I'm using an existing workload report Id, I want to make sure I don't get a collision so get a stagingin ID that doesn;t exist

Reply:
Oh, missed that bit. Well in that case, why are you not inserting a workload report? If we run this on an environment with no workload reports seeded, the tests will fail?